src/plugins/ui_stateful/clipboard.ts

```diff
@@ -109,6 +109,46 @@ export class ClipboardPlugin extends UIPlugin {
           this.status = "invisible";
         }
         break;
+      case "COPY_PASTE_CELLS_ABOVE":
+        if (this.getters.getSelectedZones().length === 1) {
+          const zone = this.getters.getSelectedZone();
+          if (zone.top === 0 && zone.bottom === 0) {
```

Comment:
this is typically something that we should reject in the allowDispatch rather than having a command that succeeds but does nothing

Reply:
Absolutely! Applied the changes :+1: